documentation/Installing-Clear-Containers-on-RHEL.md

```diff
@@ -1,206 +1,58 @@
-# Install the Clear Containers runtime on RHEL 7.3
+# Install Intel® Clear Containers runtime on RHEL 7.3
 
+## Required Setup
 
-## Introduction
-
-Installing Clear Containers on RHEL is slightly more difficult than on other distributions mainly due to the fact that it does not provide current versions of `gcc` and `glibc`.
-
-Hence, this guide explains how to install updated packages in `/usr/local`, to avoid conflicting with the base packages.
-
-## Warning
-
-Note carefully the comment in the section [Install container kernel and rootfs](#install-container-kernel-and-rootfs). You MUST update the packages listed regularly to ensure your Clear Containers benefit from performance improvements, new features and most importantly CVE security fixes.
-
-## Install updated glib and json-glib
-
-Note: these packages need to be built with the default RHEL compiler since they don't build by default with `gcc` 6.
+If you are not using sudo, please ensure that your current user can run sudo without specifying a password:
```

Comment:
The installation requires the current user to run sudo without specifying a password. Verify this with the following commands: